tests/fuzzing/oss_fuzz_build.sh

```diff
@@ -8,6 +8,15 @@
 # OSS-fuzz environment.
 # More info about compile_go_fuzzer() can be found here:
 #     https://google.github.io/oss-fuzz/getting-started/new-project-guide/go-lang/#buildsh
+
+if [[ $SANITIZER = *coverage* ]]; then
+        compile_go_fuzzer github.com/opencontainers/runc/libcontainer/system FuzzUIDMap id_map_fuzzer linux gofuzz
+        compile_go_fuzzer github.com/opencontainers/runc/libcontainer/user FuzzUser user_fuzzer gofuzz
+        compile_go_fuzzer github.com/opencontainers/runc/libcontainer/configs FuzzUnmarshalJSON configs_fuzzer gofuzz
+
+        exit 0
+fi
```

Comment:
It seems to me that it might be simpler to have some `$fuzz_type` variable which you then use like this:

```
fuzz_type=
if [[ $SANITIZER = *coverage* ]];
then
    fuzz_type=gofuzz
fi

compile_go_fuzzer github.com/opencontainers/runc/libcontainer/system FuzzUIDMap id_map_fuzzer linux $fuzz_type
compile_go_fuzzer github.com/opencontainers/runc/libcontainer/user FuzzUser user_fuzzer $fuzz_type
compile_go_fuzzer github.com/opencontainers/runc/libcontainer/configs FuzzUnmarshalJSON configs_fuzzer $fuzz_type
```

Reply:
We also call a different path when sanitizer is set to "coverage".

At the moment this is a slight workaround. We are working on the OSS-fuzz side to use a single `compile_go_fuzzer` for both cases.

Reply:
Fair enough (though I don't see why the import path can't be the same in both cases).